doc/source/reference/c-api/array.rst

```diff
@@ -819,38 +820,42 @@ General check of Python Type
     that looks for the :obj:`~numpy.class.__array__` attribute. `context` is
     unused.
 
-.. c:function:: PyArray_IsZeroDim(op)
+.. c:function:: int PyArray_IsZeroDim(PyObject *op)
 
     Evaluates true if *op* is an instance of (a subclass of)
     :c:data:`PyArray_Type` and has 0 dimensions.
 
-.. c:function:: PyArray_IsScalar(op, cls)
+.. c:var:: op
+
+.. c:var:: cls
```

Comment:
Why are these needed?

Reply:
sphinx 2 doesn't support the function style macro, so emits reference warnings without these vars. we don't need these in sphinx 3.

Reply:
Since this change leads to confusing rendering in the docs, I'd be inclined to save this bit for the Sphinx 3 PR.